### PR TITLE
RELEASE v.0.7.2

### PR DESCRIPTION
**Pull request by mmcky**
This PR sets up a new version `v0.7.2`

- [x] update changelog
- [x] update version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## Ver 0.7.2 (14th-March-2024)
+
+## What's Changed
```
Comment (oyamad) [inline CHANGELOG.md:5]:
```suggestion
### Fix
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+
+* FIX: linprog_simplex: Fix bug in Phase 1 by @oyamad in https://github.com/QuantEcon/QuantEcon.py/pull/727
+
+## Maintenance
```
Comment (oyamad) [inline CHANGELOG.md:9]:
```suggestion
### Maintenance 
```

**Review by oyamad — COMMENTED** [review]
Just to be consistent with other entries...

**Comment by mmcky** [comment]
> Just to be consistent with other entries...

thanks @oyamad I was thinking of transitioning to the auto-generated output from github. But I agree I think maintaining consistency is better. 👍

**Comment by mmcky** [comment]
@oyamad once I see green ticks I'll merge and release. thanks for the ping.

**Review by oyamad — APPROVED** [review]
Thanks!

**Pull request merged**